packages/wonder-blocks-core/components/with-ssr-placeholder.js

```diff
@@ -129,6 +129,31 @@ export default class WithSSRPlaceholder extends React.Component<Props, State> {
         const {children, placeholder} = this.props;
 
         switch (renderState) {
+            // There are edge cases where for some reason, we get an unknown
+            // context value here. So far it seems to be when we're nested in a
+            // v1 WithSSRPlaceholder equivalent component, or in some older
+            // React v16 situations where we're nested in the provider of a
+            // different context.
```

Comment:
I thought it was an issue specifically with apollo and react-router's abuse of Context API internals.  IS that no longer the case?

Reply:
I don't know for sure. That's merely a guess.

Reply:
I have not been able to repro outside of webapp nor identify why webapp is bleeding values across contexts. All thoughts on why it happens are really guesses. I've made this stuff more robust to that circumstance and tested it out with Craig's changes to check that it address the core issues.

Why the context bleeds at all is yet to be fully understood.